capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -18,7 +18,74 @@
 function createMap() { // eslint-disable-line no-unused-vars
   const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
+      {center: {lat: 39.828502, lng: -98.579512},
+        zoom: 4,
+      },
+  );
+
+  // Set custom color scheme for the map.
+  const styles =
+    [
+      {
+        'featureType': 'administrative',
+        'elementType': 'labels.text.fill',
+        'stylers': [
+          {
+            'color': '#000000',
```

Comment:
nit: #000

Reply:
It needs to be an RGB hex string of format #RRGGBB. [ref](https://developers.google.com/maps/documentation/javascript/style-reference#stylers)

Reply:
does it not work with #000? Whenever possible, you should try using [#rgb](https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation). If it doesn't work then that's fine

Reply:
@Leoism yeah I tried it with #000 and it didn't work

Reply:
Okay, that's fine!